docs/queryworkers/akamai/generate-edge-worker-stream.md

```diff
@@ -0,0 +1,17 @@
+---
+sidebar_position: 25
+title: Generate Edge Worker from Stream Worker
+---
+
+Generate an edge worker from a Stream worker.
+
+1. [Log in to your Macrometa account](https://auth.paas.macrometa.io/).
+1. Click **Stream Workers**.
+1. Select a stream worker or [create a new one](../../cep/index.md).
```

Comment:
When we actually write a task for creating a stream worker, let's update this link.